packages/lib/src/Button.wc.svelte

```diff
@@ -22,27 +16,28 @@
     lg: "px-6 py-3 text-lg",
   };
 
-  const variantClasses = {
-    primary: {
-      solid:
-        "bg-[var(--sesamy-button-bg)] text-[var(--sesamy-button-text)] hover:bg-[var(--sesamy-button-bg-hover)] focus:ring-[var(--sesamy-button-ring)]",
-      outline:
-        "bg-transparent border border-[var(--sesamy-button-border)] text-[var(--sesamy-button-text)] hover:bg-[var(--sesamy-button-bg-hover)] hover:text-[var(--sesamy-button-text-hover)] focus:ring-[var(--sesamy-button-ring)]",
-    },
-    secondary: {
-      solid:
-        "bg-[var(--sesamy-button-secondary-bg)] text-[var(--sesamy-button-secondary-text)] hover:bg-[var(--sesamy-button-secondary-bg-hover)] focus:ring-[var(--sesamy-button-secondary-ring)]",
-      outline:
-        "bg-transparent border border-[var(--sesamy-button-secondary-border)] text-[var(--sesamy-button-secondary-text)] hover:bg-[var(--sesamy-button-secondary-bg-hover)] hover:text-[var(--sesamy-button-secondary-text-hover)] focus:ring-[var(--sesamy-button-secondary-ring)]",
-    },
+  const getButtonClasses = (variant: Variant) => {
+    switch (variant) {
+      case "primary":
+        return "bg-blue-500 text-white hover:bg-blue-600 focus:ring-blue-300";
+      case "secondary":
+        return "bg-transparent border border-blue-500 text-blue-500 hover:bg-blue-50 hover:text-blue-600 focus:ring-blue-300";
+      case "tertiary":
+        return "text-blue-500 hover:text-blue-600 focus:ring-blue-300";
+    }
```

Comment:
What's the reason for doing these as functions rather than inline? I would argue this is non-conventional

Reply:
You mean like it was before? Yeah.. think that might be better. I've been changing things back and forth and didn't really reflect on it. I'll change it

Reply:
Changed it back to the old format.